src/Services/MeilisearchService.php

```diff
@@ -221,6 +222,21 @@ public function shouldBeIndexed(object $entity): bool
         return true;
     }
 
+    private function getBaseClassName($object_or_class): string
```

Comment:
```suggestion
    /**
     * @param object|class-string $objectOrClass
     */
    private function getBaseClassName(object|string $objectOrClass): string
```

Reply:
Union type is not allowed in PHP 7.4

Reply:
Ok so at least phpdoc and camelcased param